src/ecalc/libraries/libecalc/common/tests/expression/test_expression_evaluator.py

```diff
@@ -93,6 +93,12 @@ def test_lexer():
     token_values = [token.value for token in tokens]
     assert token_values == compare
 
+    expression = "2.4e6 {/} 100 {*} 1.35e-05"
```

Comment:
I would make a separate test. I see that we have fewer tests here than anticipated. Make sure you cover all edge cases. A good idea is to add parametric tests (my preference) to test both the case with scientific numbers alone, in an expression with other scientific numbers and combined with other tokentypes :)

Reply:
Have updated with more parametric tests now